### PR TITLE
Restored breadcrumbs back to Pod from Container details page

### PR DESCRIPTION
**Pull request by dtaylor113**
image

image
image

image

Fix for: https://jira.coreos.com/browse/CONSOLE-1204

### CODE REVIEW

File: frontend/public/components/container.jsx [hunk 1]
```diff
@@ -251,21 +251,21 @@ const Details = (props) => {
...
     kind: 'Pod',
     isList: false,
     prop: 'obj',
   }]}>
+    <PageHeading
+      detail={true}
+      title={props.match.params.name}
+      kind="Container"
+      breadcrumbsFor={() => [
+        {name: props.match.params.podName, path: `${props.match.url.split('/').filter((v, i) => i <= props.match.path.split('/').indexOf(':podName')).join('/')}`},
```
Comment (spadgett) [inline frontend/public/components/container.jsx]: I'd rather avoid path math and just build the URL for a pod.

**Comment by spadgett** [comment]
/assign
@jwforres fyi

File: frontend/public/components/container.jsx (hunk 1 above)
Comment (dtaylor113) [reply frontend/public/components/container.jsx]: Now using `resourcePath(..)`

Comment (spadgett) [reply frontend/public/components/container.jsx]: Yeah I think this is cleaner, thanks. (I realize you had just moved the existing code previously.)

**Comment by spadgett** [comment]
jenkins rebuild

**Pull request merged**